Jenkinsfile

```
@@ -27,12 +27,19 @@ def confluent_release_quality = choice(name: 'CONFLUENT_RELEASE_QUALITY',
     description: 'Determines the release extention (suffix) (ie: "prod" for public releases, "snapshot" for nightly builds)',
 )
 
+// Parameter for the molecule test scenario to run
+def molecule_scenario_name = choice(name: 'SCENARIO_NAME',
+    choices: ['plaintext-rhel', 'customcerts-rhel'],
```

```suggestion
    choices: ['plaintext-rhel'],
```

I won't offer it for now then - Its here as a place holder for the newer branches that have more scenarios.